subsys/net/lib/dns/resolve.c

```diff
@@ -92,14 +92,17 @@ int dns_resolve_init(struct dns_resolve_context *ctx, const char *servers[])
 	int ret, count;
 
 	if (!ctx) {
+    printk("\nMad about context\n\n");
```

Comment:
very mad... 
was this for debugging?

Reply:
@nashif you move faster than I can Type WIP :)